src/USER-LD/pair_localdensity.h

```diff
@@ -0,0 +1,88 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+-------------------------------------------------------------------------
+   pair_localdensity written by:
+   Tanmoy Sanyal and M. Scott Shell from UC Santa Barbara
+   David Rosenberger: TU Darmstadt
+-------------------------------------------------------------------------*/   
+    
+
+#ifdef PAIR_CLASS
+
+PairStyle(localdensity,PairLOCALDENSITY)
```

Comment:
for improved consistency with the rest of LAMMPS, please name the pair style `local/density`, the class `PairLocalDensity` and source and documentation files `pair_local_density.<ext>`